frontend/src/components/ErrorBoundary/useErrorReset.ts

```
@@ -0,0 +1,99 @@
+ *   useErrorReset(
+ *     () => setHasError(false),
+ *     hasError // Only reset when there's an error
+ *   );
+ *   
+ *   // ... rest of error boundary logic
+ * }
+ * ```
+ */
+export function useErrorReset(
+  resetFn: () => void,
+  enabled: boolean = true
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)